conf.d/nvm.fish

```diff
@@ -1,9 +1,12 @@
+set -q XDG_CONFIG_HOME; or set XDG_CONFIG_HOME ~/.config
+set -g nvm_bin_path $XDG_CONFIG_HOME/nvm/bin
+
+if not contains $nvm_bin_path $PATH
+    set PATH $nvm_bin_path $PATH
+end
```

Comment:
It seems if we'd move this into `nvm` function itself (with appropriate configuration checking), it should also fix https://github.com/jorgebucaran/fish-nvm/issues/101

Reply:
I could look into that, but @jorgebucaran has not made any comments so I am not sure if he has any intention to merge this.